.github/workflows/main.yml

```diff
@@ -0,0 +1,40 @@
+name: testcontainers-python
+on:
+  - push
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        python-version: [3.5, 3.6, 3.7, 3.8]
+    runs-on: ubuntu-16.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
```

Comment:
We need to supply the version information here for the setup script to pick things up. Cf. https://github.com/tillahoffmann/idxhound/blob/master/.github/workflows/pythonpackage.yml#L25-L26

Reply:
thanks for pointing that out. I can't believe I missed that.

Updated the code.